### DISCUSSION TITLE
[Feature] Multi-select on /search results

### DISCUSSION THREAD

**Discussion by SkepticMystic** [immich-app/immich#12947] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

On the /photos page, I can `Shift + Click` to multiselect many photos at once. However, on the /search results page, this functionality isn't available. Multiple photos can be selected, but only by clicking each one at a time.

This applies to the Web platform. Multi-select _is_ possible on the mobile /search results page.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#9159

**Reply by SkepticMystic** [reply]
Thank you! My search didn't pick that up
